### ISSUE TITLE
Encrypted VHF/UHF Key IDs Missing

### ISSUE THREAD

**Issue by Trunkmaan** [s2underground/GhostNet#27]
All radios must have the same keys with the same key identifiers. Loading radios with the same key but with different key identifiers will result in failed decryption.

**Comment by s2underground** [comment]
Yeah you're absolutely right, I forgot that. Until i can get 1.6 finished up, let's use Key 1, which is what I'm guessing most people will use if they are just getting started.

**Comment by Trunkmaan** [comment]
Sounds good. Key 1 for VHF and key 2 for UHF should do for now.
